guides/source/5_0_release_notes.md

```diff
@@ -52,10 +52,35 @@ choice.
 See the [Action Cable Overview](action_cable_overview.html) guide for more
 information.
 
-### Rails API
-[Pull Request](https://github.com/rails/rails/pull/19832)
+### API Applications
 
-ToDo...
+Rails can now be used to create slimmed down API only applications.
+This is useful for creating and serving APIs similar to [Twitter](https://dev.twitter.com) or [GitHub](http://developer.github.com) API, 
+that can be used to serve public facing as well as for own custom Apps.
```

Comment:
Can you change the following:
"that can be used to serve public facing, as well as, for custom apps."